### PR TITLE
Feature/request blood

### PR DESCRIPTION
**Pull request by tia-gijo**
Hey

### CODE REVIEW

File: src/main/resources/static/requestBlood.html [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+        alert("Fields cannot be empty")
+        console.log("fields are empty");
+    }
+
+}
+
+</script>
+
+
+<script>
```
Comment (Kyle-WSU) [inline src/main/resources/static/requestBlood.html:262]: is there a reason why you made to separate script blocks?

File: src/main/resources/static/requestBlood.html [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+        document.getElementById("name").focus();
+        check = false;
+  }
+ 
+
+  return check;
+}
+</script>
+
+<script>
```
Comment (Kyle-WSU) [inline src/main/resources/static/requestBlood.html:307]: again is there a reason why you split all these script blocks?

File: src/main/resources/static/requestBlood.html [hunk 3]
```diff
@@ -0,0 +1,315 @@
+
```
Comment (Kyle-WSU) [inline src/main/resources/static/requestBlood.html:1]: shouldn't this page be restricted to verified users? If so, the login button and modal can be deleted

**Review by Kyle-WSU — CHANGES_REQUESTED** [review]

File: src/main/resources/static/requestBlood.html (hunk 1 above)
Comment (tia-gijo) [reply src/main/resources/static/requestBlood.html:262]: will fix it

File: src/main/resources/static/requestBlood.html (hunk 3 above)
Comment (tia-gijo) [reply src/main/resources/static/requestBlood.html:1]: that page is the old request blood, the new one is in the template folder

**Review by TheKPham — APPROVED** [review]

**Pull request closed without merging**